### PR TITLE
squashed some bugs

### PR DESCRIPTION
**Pull request by Teknicl**

### CODE REVIEW

File: flask_app/controllers/projects.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from crypt import methods
```
Comment (Teknicl) [inline flask_app/controllers/projects.py:1]: @aacole9 I'm thinking you were going to add the bcyrpt library here but not sure we need it on non user tables.

File: flask_app/controllers/tasks.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from crypt import methods
```
Comment (Teknicl) [inline flask_app/controllers/tasks.py:1]: @aacole9 I'm thinking you were going to add the bcyrpt library here but not sure we need it on non user tables.

File: flask_app/controllers/tasks.py [hunk 3]
```diff
@@ -12,14 +12,14 @@
 def post_task():
     return render_template("post.html")
 
-@app.route('/dashboard')
```
Comment (Teknicl) [inline flask_app/controllers/tasks.py:15]: @aacole9 this route was duplicated so I commented this one out in favor of the controller project dashboard route.

**Review by Lopez-CL — COMMENTED** [review]
Uh oh there are a few file changes that I made that are being overwritten. Mainly some variables in returns. I think it's fine, they were small changes.

**Review by Lopez-CL — APPROVED** [review]

**Comment by Teknicl** [comment]
> Uh oh there are a few file changes that I made that are being overwritten. Mainly some variables in returns. I think it's fine, they were small changes.

To get the code working I commented out what Flask couldn't interpret.  They looked like future variables but Flask was screaming at them when you run the server.  Hope you are able to easily uncomment and add back into the return statement.

**Comment by Lopez-CL** [comment]
Ahh, i see now. I think mistook those comments as the comments I left for Adam. That all sounds good!

**Pull request merged**